github/helpers.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Data structures to assist with reporting mis-configured items
+
+from dataclasses import dataclass
+from typing import Any, ClassVar, List, Optional
+
+
+@dataclass
+class Criteria:
```

Comment:
Would love to see a docstring for this class.

Reply:
coming post refactor